core/services/offchainreporting2/data_source.go

```diff
@@ -29,6 +29,24 @@ type dataSource struct {
 
 var _ median.DataSource = (*dataSource)(nil)
 
+func NewDataSource(
```

Comment:
if you rebase I pulled this into a shared ocr1/2 data_source file

Reply:
Yeah, rebased on top of Aaron's branch and using his updated data source from ocrcommon.